routes/drinks.js

```diff
@@ -0,0 +1,7 @@
+const express = require('express');
+const router = express.Router();
+const { getMainPage } = require('../controllers/drinks');
+
+router.get('/mainpage/:quantity', getMainPage);
```

Comment:
remove quantity and use query params:
replace `router.get('/mainpage/:quantity', getMainPage);` -->

`router.get('/mainpage', getRandomDrinks);`